packages/rancher-istio-1.8/charts/templates/istio-psp.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if .Values.global.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: istio-psp
+  namespace: {{ template "istio.namespace" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: istio-psp
+subjects:
+  - kind: ServiceAccount
+    name: istio-egressgateway-service-account
+  - kind: ServiceAccount
+    name: istio-ingressgateway-service-account
+  - kind: ServiceAccount
+    name: istio-mixer-service-account
+  - kind: ServiceAccount
+    name: istio-operator-authproxy
+  - kind: ServiceAccount
+    name: istiod-service-account
+  - kind: ServiceAccount
+    name: istio-sidecar-injector-service-account
+  - kind: ServiceAccount
+    name: istiocoredns-service-account
+  - kind: ServiceAccount
+    name: default
```

Comment:
Are these from somewhere ? How will we know when a new one is added or existing one updated ?

Reply:
Each release we will have to determine if a new one was added to the manifests or not. These were pulled from what is being installed in the newer istio versions.

Reply:
Ok, we need to add this to a checklist or something to ensure it gets reviewed regularly, definitely with minor releases.